samcli/lib/translate/translate_utils.py

```diff
@@ -0,0 +1,31 @@
+"""
+Library for Validating Sam Templates
+"""
+import os
+import logging
+import click
```

Comment:
I believe we should not use click outside of the `commands` directory: https://github.com/aws/aws-sam-cli/blob/develop/DEVELOPMENT_GUIDE.md#code-conventions

Reply:
Moved it to the commands directory